main.go

```diff
@@ -61,6 +63,10 @@ type tlsConfig struct {
 	cipherSuites []string
 }
 
+type Configfile struct {
```

Comment:
nit: do we need to make this public? For json parsing only this can be left private.

Reply:
No sorry actually it was not exported at first and I was debugging something. Will change to be unexported.